CMakeLists.txt

```diff
@@ -133,7 +133,7 @@ endif()
 
 include(${HPCC_SOURCE_DIR}/cmake_modules/optionDefaults.cmake)
 
-if ( REMBED )
+if ( REMBED AND (NOT INCLUDE_PLUGINS) )
   # various components that are not needed if only making the R plugin
   SET(USE_LIBARCHIVE 0)
   SET(USE_APR 0)
```

Comment:
For no plugin htpasswd component is already included. The fix just make if "-DINCLUDE_PLUGINS=ON" htpasswd is also built.
So you think building system/security/plugins/htpasswd doesn't fix the problem?